.happy/config.json

```diff
@@ -3,7 +3,7 @@
     "terraform_version": "0.13.5",
     "default_env": "rdev",
     "app": "hosted-cellxgene",
-    "default_compose_env": ".env.ecr",
+    "default_compose_env_file": ".env.ecr",
```

Comment:
Is this a happy golang related change or was it just broken? If the former, do we also need to update config_version to v2? (per [these docs](https://czi.atlassian.net/wiki/spaces/SI/pages/2429288766/Golang+Happy+CLI#Migrating-existing-repos-to-Golang-Happy-CLI))

Reply:
>If the former, do we also need to update config_version to v2? (per [these docs](https://czi.atlassian.net/wiki/spaces/SI/pages/2429288766/Golang+Happy+CLI#Migrating-existing-repos-to-Golang-Happy-CLI))
we will do those once we've migrated